viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingController.java

```diff
@@ -3089,12 +3086,11 @@ public void commonNewDocumentHandling(String fileDescription) {
         // add to the main pdfContentPanel the document peer
         if (viewer != null) {
             File f = new File(fileDescription);
-            String argument = f.exists() ? f.getName() : fileDescription;
-            Object[] messageArguments = {argument};
-            MessageFormat formatter = new MessageFormat(
-                    messageBundle.getString("viewer.window.title.open.default"));
+            String title = document.getInfo().getTitle();
```

Comment:
You'll need to add some protection for document.getInfo(), there are cases where this can be null, similar for th summary Frame.  Interesting you added this when I was playing around with this PR a while back I did the same thing with the title formating.  Anyways looks good otherwise.